ledger/participant-integration-api/src/main/scala/platform/store/backend/localstore/PartyRecordStorageBackendImpl.scala

```diff
@@ -105,14 +105,30 @@ object PartyRecordStorageBackendImpl extends PartyRecordStorageBackend {
     )
   }
 
-  override def updateIdentityProviderId(
-      internalId: Int,
+  override def fetchPartiesExist(
```

Comment:
maybe
```suggestion
  override def fetchPartiesForExistingPartyRecords(
```
or
```suggestion
  override def filterExistingParties(
```
or
```suggestion
  override def fiterPartiesForExistingPartyRecords(
```
?